### PR TITLE
Fix invisible Course name setting

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-14034
affects: Teacher
release note: Fixed an issue with course settings hiding the name

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: rn/Teacher/src/modules/courses/settings/CourseSettings.js [hunk 1]
```diff
@@ -227,6 +228,12 @@ const styles = createStyleSheet((colors, vars) => ({
     color: colors.textDarkest,
     lineHeight: 54,
   },
+  inputText: {
+    flex: 3,
+    color: colors.textDarkest,
+    fontSize: 16,
+    textAlign: 'right',
```
Comment (vanwagonet) [inline rn/Teacher/src/modules/courses/settings/CourseSettings.js]: the `lineHeight` property was making the input ridiculously big & the text was clipped below the visible space. This works around the awful bug in RN.

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/4b5219i) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F4b5219i)
teacher: [Link](https://rebrand.ly/znwy5e9) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fznwy5e9)
parent: [Link](https://rebrand.ly/lfz441k) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Flfz441k)

**Comment by inst-danger** [comment]
#### Release Note: 
Fixed an issue with course settings hiding the name

#### Affected Apps: Teacher

[MBL-14034](https://instructure.atlassian.net/browse/MBL-14034)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.63% | 93.63% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: rn/Teacher/src/modules/courses/settings/CourseSettings.js [hunk 2]
```diff
@@ -137,7 +137,7 @@ export class CourseSettings extends Component<Props, State> {
                 </Text>
                 <TextInput
                   value={this.state.name}
-                  style={styles.actionableText}
```
Comment (bluwave) [inline rn/Teacher/src/modules/courses/settings/CourseSettings.js]: do we need to cleanup actionableText or it's still used?

Comment (vanwagonet) [reply rn/Teacher/src/modules/courses/settings/CourseSettings.js]: It's still used for a different row.

**Pull request merged**